### DISCUSSION TITLE
Task, project complete or closed check box

### DISCUSSION THREAD

**Discussion by stramag** [te-online/timemanager#98] [Ideas]
Dear Thomas,

I tried your application and it looks very helpful to me,
But i didn't see any solution to how can i handle the closed project and tasks.

Please let me know if this is already implemented into this application.

Thank you for your help!

Best Regards,
GáborS

**Comment by te-online** [comment]
Hey there 👋 

Thanks for your question. Have a look at this issue: https://github.com/te-online/timemanager/issues/38

I think the person is looking for the same thing. Unfortunately, it's not there right now 😢 
But you can help adding it, if you'd like to :-)
